Sources/ObjCSupport/LOKLayoutArrangementSection.swift

```diff
@@ -10,7 +10,9 @@ import Foundation
 
 @objc open class LOKLayoutArrangementSection: NSObject {
     let unwrapped: Section<[LayoutArrangement]>
+    @objc public let items: [LOKLayoutArrangement]
```

Comment:
please add similar properties for header and footer as well

Reply:
Done